src/Microsoft.AspNetCore.Mvc.Core/Internal/MvcCoreMvcOptionsSetup.cs

```diff
@@ -23,7 +24,7 @@ public class MvcCoreMvcOptionsSetup : IConfigureOptions<MvcOptions>
         private readonly ILoggerFactory _loggerFactory;
 
         public MvcCoreMvcOptionsSetup(IHttpRequestStreamReaderFactory readerFactory)
-            : this(readerFactory, loggerFactory: null)
+            : this(readerFactory, NullLoggerFactory.Instance)
```

Comment:
If this ctor is only used in tests, add a comment.

Why make this change? The changed binders and providers could handle `null` the same way `BodyModelBinder` / `BodyModelBinderProvider` do.

Reply:
I'm missing something. Being inconsistent with `BodyModelBinder` / `BodyModelBinderProvider` today and changing what that pair do is consistent with what?